### ISSUE TITLE
Reproduction and lmms-eval Differences in VSI-Bench Results

### ISSUE THREAD

**Issue by TUe-Ray** [LaVi-Lab/VG-LLM#66]
Thank you for the amazing work !!!

I have two questions regarding the VSI-Bench results:

1. From your HuggingFace eval.log, I saw that the reported vsibench_score is 50.6616 (rounded to 50.7 in the paper). May I ask which lmms-eval git hash or version was used for that evaluation, if you happen to remember?
When I re-ran the evaluation using your released pretrained model and the same configuration as in your eval.sh script (32 frames, greedy decoding, 0-shot, full test split), I obtained a score of 50.8634. I am wondering whether this small difference could be due to lmms-eval version differences or possible dataset revisions.

<img width="1209" height="149" alt="Image" src="https://github.com/user-attachments/assets/5e97e681-4115-4979-a46b-b1acc95289b3" />

2. I also trained the model following the same training setup described in your work in order to reproduce the results. However, under the same evaluation configuration, my reproduced model achieves 49.35 on VSI-Bench. This gap (around 1.3–1.5 points) seems slightly larger than what I would expect from pure training randomness.  I am using cross-node distributed training, but other than that, all settings (batch size, learning rate, scheduler, number of frames, seed, etc.) are kept identical to your released configuration.
In your experience, are there any specific factors that are particularly sensitive for this benchmark that might lead to such a difference?

<img width="1192" height="136" alt="Image" src="https://github.com/user-attachments/assets/ece26777-2238-4d61-9ebc-7cedd74184fd" />

Thank you very much for your time and for sharing your work!

**Comment by CSPlayer0689** [comment]
> Thank you for the amazing work !!!
> 
> I have two questions regarding the VSI-Bench results:
> 
> 1. From your HuggingFace eval.log, I saw that the reported vsibench_score is 50.6616 (rounded to 50.7 in the paper). May I ask which lmms-eval git hash or version was used for that evaluation, if you happen to remember?
>    When I re-ran the evaluation using your released pretrained model and the same configuration as in your eval.sh script (32 frames, greedy decoding, 0-shot, full test split), I obtained a score of 50.8634. I am wondering whether this small difference could be due to lmms-eval version differences or possible dataset revisions.
> 
> <img alt="Image" width="1209" height="149" src="https://private-user-images.githubusercontent.com/180642862/555056187-5e97e681-4115-4979-a46b-b1acc95289b3.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzcxMjAzOTYsIm5iZiI6MTc3NzEyMDA5NiwicGF0aCI6Ii8xODA2NDI4NjIvNTU1MDU2MTg3LTVlOTdlNjgxLTQxMTUtNDk3OS1hNDZiLWIxYWNjOTUyODliMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwNDI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDQyNVQxMjI4MTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wODYyODI0OWY4ZGJmZTVkZGVjMTFhODc2ODQ1NzAzNmEzNGJlMGZlYTlkYjExMjkwNGU2YWE2YzFiMjMxZTU3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZyZXNwb25zZS1jb250ZW50LXR5cGU9aW1hZ2UlMkZwbmcifQ.0Ki1iUGY9LJ7N5S_wXgIBxCaB_10_8WY9IVECrxBjJ4">
> 2. I also trained the model following the same training setup described in your work in order to reproduce the results. However, under the same evaluation configuration, my reproduced model achieves 49.35 on VSI-Bench. This gap (around 1.3–1.5 points) seems slightly larger than what I would expect from pure training randomness.  I am using cross-node distributed training, but other than that, all settings (batch size, learning rate, scheduler, number of frames, seed, etc.) are kept identical to your released configuration.
>    In your experience, are there any specific factors that are particularly sensitive for this benchmark that might lead to such a difference?
> 
> <img alt="Image" width="1192" height="136" src="https://private-user-images.githubusercontent.com/180642862/555056933-ece26777-2238-4d61-9ebc-7cedd74184fd.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzcxMjAzOTYsIm5iZiI6MTc3NzEyMDA5NiwicGF0aCI6Ii8xODA2NDI4NjIvNTU1MDU2OTMzLWVjZTI2Nzc3LTIyMzgtNGQ2MS05ZWJjLTdjZWRkNzQxODRmZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwNDI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDQyNVQxMjI4MTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xZTJjZTA3MTgwMmYzOTMxNmIwYmMzZGVhNDZhMDI1ODhlMDdmMWRhMTg0NDAwZWI0NjcxNzZjZTA1ZmUyMTNjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZyZXNwb25zZS1jb250ZW50LXR5cGU9aW1hZ2UlMkZwbmcifQ.1hWFsKm8YqHX1DGzKgR517Nz06p3iOt3kDc4xjiaPe0">
> Thank you very much for your time and for sharing your work!

Hi, I also observed similar  gap when I train the model several times with the same config setting. May I ask if you find the reason?
